realm/realm-jni/src/io_realm_internal_tableview.cpp

```diff
@@ -131,6 +131,18 @@ JNIEXPORT void JNICALL Java_io_realm_internal_TableView_nativeClose(
     delete TV(nativeViewPtr);
 }
 
+JNIEXPORT jlong JNICALL Java_io_realm_internal_TableView_nativeGetCopy(
+    JNIEnv* env, jobject, jlong nativeViewPtr)
+{
+    if (!VIEW_VALID_AND_IN_SYNC(env, nativeViewPtr))
+        return 0;
+    try {
+        TableView* copiedTableView = new TableView(*TV(nativeViewPtr));
```

Comment:
@kneth yes, it performs a copy which gives you a new valid independent tableview, while keeping the original tableview valid and independent as well. So kindof a deep copy.

However, just note that the two tableviews can only be used from the same thread.